lte/gateway/c/core/oai/test/amf/test_amf_procedures.cpp

```diff
@@ -339,4 +357,345 @@ TEST_F(AMFAppProcedureTest, TestPDUSessionSetup) {
   EXPECT_TRUE(rc == RETURNok);
 }
 
+TEST_F(AMFAppProcedureTest, TestPDUSessionFailure_dnn_not_subscribed) {
+  int rc                 = RETURNerror;
```

Comment:
Use enum type instead of int
```suggestion
status_code_e rc                 = RETURNerror;
```

Reply:
All UT and AMF api interfaces are not using  status_code_e enum for return value and returning as int. was a major code change if we need to change all.
Need to take separate activity to resolve all.